src/peft/peft_model.py

```diff
@@ -650,6 +677,84 @@ def _split_kwargs(cls, kwargs: Dict[str, Any]):
 
         return hf_hub_download_kwargs, other_kwargs
 
+    def _update_offload(self, offload_index: Dict[Dict[str:str]], adapters_weights: Dict[str : torch.tensor]):
+        """
+        Update the offload_index and safetensors files for loading and mergine PeftModels with disk-offloaded modules.
+
+        Args:
+            offload_index (Dict[str: str]):
+                Dictionary of disk-offloaded modules with their metadata and safetensors filenames
+            adapters_weights (Dict[str: torch.tensor]):
+                Dictionary of Peft adapter module names and weights
+        """
+
+        prefix = "base_model.model."
+        # rename offload index weight and model names
+        adapter_names = list(self.peft_config.keys())
+        for adapter_name in adapter_names:
+            target_modules = self.peft_config[adapter_name].target_modules
+            if offload_index:
```

Comment:
So IIUC, if we don't have `offline_index`, nothing really needs to happen. So can we add

```python
if not offload_index:
    return offload_index
```

to the start of the function and remove the `if offload_index:` check here? This should make the code easier to read and save one level of indentation.

Reply:
Very true, and it does make everything a bit more readable